src/common/reactNativeProjectHelper.ts

```diff
@@ -13,8 +14,8 @@ export class ReactNativeProjectHelper {
         return ["0.54.0", "0.54.1", "0.54.2", "0.54.3", "0.54.4"];
     }
 
-    public static getReactNativeVersion(projectRoot: string) {
-        return new CommandExecutor(projectRoot).getReactNativeVersion();
+    public static getReactNativeVersion(projectRoot: string, logger?: ILogger) {
```

Comment:
Please remove logger from `getReactNativeVersion` function, we can add logging based on the results of this function.